<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ubuntu Sticky Notes</string>
  </property>

  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="search_bar">
      <property name="placeholderText">
       <string>Search...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QListWidget" name="list_widget"/>
    </item>
   </layout>
  </widget>

  <widget class="QToolBar" name="main_toolbar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <addaction name="new_action"/>
   <addaction name="bin_action"/>
  </widget>

  <action name="new_action">
   <property name="text">
    <string>New Note</string>
   </property>
  </action>
  <action name="bin_action">
   <property name="text">
    <string>Trash</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
